catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -98,5 +107,19 @@ public void init() {
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList()),
         MAPPER::toJson);
+    delete(
+        "/forms/:id",
+        APPLICATION_JSON,
+        (req, res) -> {
+          String id = req.params(":id");
+          DeleteResponse deleteResponse = catalogFramework.delete(new DeleteRequestImpl(id));
+          if (!deleteResponse.getProcessingErrors().isEmpty()) {
+            res.status(500);
+            LOGGER.debug("Failed to delete Form " + id);
```

Comment:
this should be parameterized : `LOGGER.debug("Failed to Delete Form {}", id);`

Reply:
Updated